src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -127,6 +127,24 @@ private bool HasCallerMemberNameAttribute
                                                   && !HasCallerFilePathAttribute
                                                   && HasCallerMemberNameAttribute;
 
+        internal override AttributeAnnotations FlowAnalysisAnnotations
+        {
+            get
+            {
+                (bool memberHasAny, AttributeAnnotations annotations) = TryGetExtraAttributeAnnotations();
+                if (memberHasAny)
+                {
+                    return annotations;
```

Comment:
>return annotations [](start = 20, length = 18)

Do we have a test covering this code path?

Reply:
Debug.Assert is actually not in the follow-up PR, but the one after that. Added a tracking comment in the meantime: https://github.com/dotnet/roslyn/pull/26828/files#diff-829bf456227af3ee271c7ae37b62ffc0R7056